### PR TITLE
new changes made

### PR DESCRIPTION
**Pull request by tany2-apt**
Added informations.

### CODE REVIEW

File: tanya.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Name: Tanya Bharti
+College: IIIT Bhubaneswar
+Love travelling and Gardening! 
+
+- ✨Aptus
+## Features
+
+- Added name, college name and hobbies
+-Working as an intern @aptusdatalabs.com
```
Comment (sudh-0) [inline tanya.md:14]: Remove the `.com`

This is an example of comments in reviews.
You can make further changes to this PR by pushing to the branch in your fork with which the PR was raised @tany2-apt

**Review by sudh-0 — CHANGES_REQUESTED** [review]

**Comment by tany2-apt** [comment]
Added new file Tanya.md, made according to template.md.

**Review by sudh-0 — APPROVED** [review]

**Comment by sudh-0** [comment]
@all-contributors please add @tany2-apt for code

**Comment by sudh-0** [comment]
@all-contributors please add @tany2-apt for code

**Pull request merged**